packages/drivers/driver-base/src/documentDeltaConnection.ts

```diff
@@ -639,7 +636,7 @@ export class DocumentDeltaConnection
 		return normalizedError;
 	}
 
-	private getConnectionDetailsProps() {
+	protected getConnectionDetailsProps() {
```

Comment:
nit: wouldn't it be better to be getConnectionDetails instead of  getConnectionDetailsProps?

Reply:
This is just making it a bit explicit that it is always currently used while logging events to get the connection details related properties.